docs/source/generics.rst

```diff
@@ -223,6 +223,26 @@ example we use the same type variable in two generic functions:
    def last(seq: Sequence[T]) -> T:
        return seq[-1]
 
+Note that a variable cannot have a type variable in its type unless the type variable is bound in a containing generic class or function:
+
+.. code-block:: python
+
+    from typing import Sequence, TypeVar
+
+    T = TypeVar('T')      # Declare type variable
+
+    def first(seq: Sequence[T]) -> T:
```

Comment:
There's no need to repeat the two "good" examples -- this example should be only concerned with what doesn't work.